ENGAGEHF/Dashboard/ToDo/Views/ToDoSection.swift

```diff
@@ -0,0 +1,63 @@
+//
+// This source file is part of the ENGAGE-HF project based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import FirebaseAuth
+import FirebaseFirestore
+import SpeziViews
+import SwiftUI
+
+
+struct ToDoSection: View {
+    @State private var tasks: [ToDoItem] = []
+    @State private var state: ViewState = .idle
+    
+    @State private var tasksLoaded = false
```

Comment:
It feels like `tasksLoaded` and `state` are displaying and contain the same state information. I think we could stick with the view state setup?